androidapp/app/src/main/java/com/example/lanweather/MainActivity.kt

```diff
@@ -33,7 +39,17 @@ class MainActivity : AppCompatActivity() {
             R.id.navigation_home, R.id.navigation_forecast, R.id.navigation_settings))
 //        setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
+        println("testtestestestestestestestestestestesteststestestse")
```

Comment:
The added code here and below seem like the perfect code to stash for writing JUnit tests later on. Do you think we should do that?

Reply:
Yeah